scripts/globals/items/bottle_of_hysteroanima.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------------
+-- ID: 5262
+-- Hysteroanima
+-- Using a Hysteroanima on an Empty foe in Promyvion will prevent it from using any TP abilities for 30 seconds, similar to an Amnesia effect
+-- Abilities will appear to be readied, but will not execute. Regular attacks may still be used. 
+-----------------------------------------
+require("scripts/globals/msg")
+require("scripts/globals/status")
+require("scripts/globals/zone")
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = dsp.msg.basic.ITEM_UNABLE_TO_USE -- default result is fail
+    local family = target:getSystem()
+    local zone = target:getZoneID()
+    local validZoneList =
+        {
```

Comment:
indent still wrong

Reply:
You could also probably just use the family system Empty for it and not have to iterate.